test/test_quantized_tensor.py

```diff
@@ -181,6 +181,11 @@ def test_qtensor_permute(self):
         self.assertEqual(qr.q_zero_point(), qlast.q_zero_point())
         self.assertEqual(qlast.dequantize(), qr.dequantize())
 
+        # permuting larger tensors
+        x = torch.randn(64, 64)
+        qx = torch.quantize_per_tensor(x, 1.0, 0, torch.qint32)
+        print(qx.permute([1, 0]))
```

Comment:
I think it's better to use the `dequantize` instead of `print`:

```python
# permuting larger tensors
x = torch.randn(64, 64)
qx = torch.quantize_per_tensor(x, 1.0, 0, torch.qint32)
qx.permute([1, 0]).dequantize()
```

Reply:
oh..we can just call this function here